src/Catalog/Declination.php

```diff
@@ -144,6 +148,14 @@ public function getAffiliateLink(): ?string
     }
 
     /**
+     * Le déclinaison est-elle neuve ou d'occasion ?
```

Comment:
In english please ? :stuck_out_tongue:

Reply:
Je ne sais pas écrire en anglais :confused:

Reply:
C'est vrai que pour le coup on est dans le SDK ^^